app/assets/stylesheets/companion_window.css

```
@@ -4,6 +4,19 @@
   --color-fog-light: #f4f4f4;
 }
 
+.tooltip {
+  display: none;
+  color: #ddd;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 0.6rem;
+  background-color: #555;
```

```suggestion
  background-color: #555;
  background-color: #616161;
```
Mirador has a background color of rgba(97, 97, 97, .92) which is #616161 in hex